01_prepare_host.sh

```diff
@@ -33,7 +33,7 @@ if ! command -v kubectl 2>/dev/null ; then
 fi
 
 if ! command -v kustomize 2>/dev/null ; then
-    curl -Lo kustomize "$(curl --silent -L https://github.com/kubernetes-sigs/kustomize/releases/latest 2>&1 | awk -F'"' '/linux_amd64/ { print "https://github.com"$2; exit }')"
+    curl -Lo kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv3.2.3/kustomize_kustomize.v3.2.3_linux_amd64
```

Comment:
Can you please make the version a variable ?

Reply:
Done